schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id int,
```

Comment:
- [ ] [ _Optional_] What do you think about marking the id column as a primary key? it will make it a unique identifier for each row in this table. You can add something like
``` SQL
    PRIMARY KEY (id)
```